### PR TITLE
[FIX] edi_oca: Restore removed functions

### PR DESCRIPTION
**Pull request by etobella**
We want to keep functionality on the same version. It is marked to remove for 16

#540

### CODE REVIEW

File: edi_oca/models/edi_backend.py [hunk 1]
```diff
@@ -179,6 +179,21 @@ def _get_exchange_type_domain(self, code):
...
 
+    def _get_job_delay_params(self, exchange_record):
+        params = {}
+        channel = exchange_record.type_id.job_channel_id
+        if channel:
+            params["channel"] = channel.complete_name
+        return params
+
+    def _delay_action(self, rec):
+        # TODO: Remove this on 16.0
```
Comment (simahawk) [inline edi_oca/models/edi_backend.py:183]: Can you please log a deprecation warning?
People should switch to the new approach as soon/far as possible.

File: edi_oca/models/edi_backend.py [hunk 2]
```diff
@@ -179,6 +179,21 @@ def _get_exchange_type_domain(self, code):
             ("backend_id", "=", self.id),
         ]
 
+    def _get_job_delay_params(self, exchange_record):
+        params = {}
+        channel = exchange_record.type_id.job_channel_id
+        if channel:
+            params["channel"] = channel.complete_name
```
Comment (simahawk) [inline edi_oca/models/edi_backend.py]: here you can take values from the new exchange record method instead

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by etobella** [comment]
Comments attended

**Comment by etobella** [comment]
Comments attended

**Review by simahawk — APPROVED** [review]
thanks!

**Pull request merged**